frontend/pages/_document.tsx

```diff
@@ -6,8 +8,19 @@ export default class Document extends NextDocument {
   render() {
     return (
       <Html lang="en">
-        <Head />
-        <body>
+        <Head>
+          <link
+            href="//db.onlinewebfonts.com/c/23c0fcab84d99da0de762de7e220a6e1?family=Europa"
+            rel="stylesheet"
+          />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+            rel="stylesheet"
+          />
+          <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+        </Head>
+
+        <body style={{ overflowX: "hidden" }}>
```

Comment:
This style looks suspicious... Content shouldn't overflow, and if it does, it should be fixed. Is there a specific reason why we're using this?